cla-backend-go/config/ssm.go

```diff
@@ -78,6 +78,7 @@ func loadSSMConfig(awsSession *session.Session, stage string) Config { //nolint
 		fmt.Sprintf("cla-corporate-base-%s", stage),
 		fmt.Sprintf("cla-corporate-v1-base-%s", stage),
 		fmt.Sprintf("cla-corporate-v2-base-%s", stage),
+		fmt.Sprintf("cla-contributor-v2-base-%s", stage),
```

Comment:
You will need to set the SSM config value down below in the switch/case statement.  This is missing. As a result, the value will be empty.